plugins/package-managers/cocoapods/src/funTest/assets/projects/synthetic/subdirectory-expected-output.yml

```diff
@@ -0,0 +1,138 @@
+---
```

Comment:
The test project name is `subdirectory`, but is this the primary concern? Would it make sense to have some `npm` mention instead?

Reply:
Since this test covers both commits, I think react-native is a better name (I didn't find another `node_modules` usage with Cocoapods nor podspec files in Ruby ).